client/src/entrypoints/admin/telepath/blocks.js

```diff
@@ -43,7 +43,7 @@ function initBlockWidget(id) {
     data-value: JSON-encoded value for this block
   */
 
-  const body = document.getElementById(id);
+  const body = document.querySelector('#' + id + '[data-block]');
```

Comment:
I think it's still invalid HTML to have two elements with the same ID, so we should make it use a different ID for the long term

Reply:
It is, as per (all and latest) spec https://www.w3.org/TR/html51/dom.html#the-id-attribute
Also things like `getElementById` and `querySelector` are specced to return the first one https://dom.spec.whatwg.org/#dom-nonelementparentnode-getelementbyid / https://dom.spec.whatwg.org/#dom-parentnode-queryselector